Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ghcr.io/cucapra/calyx:0.4.0
+
+WORKDIR /home
+ADD . filament
+# Build the compiler
+WORKDIR /home/filament
+RUN CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse cargo build --all
```

Comment:
Might be worth installing `z3` and `cvc5` and setting up the `fud` stage like we do in the github actions: https://github.com/cucapra/filament/blob/main/.github/workflows/test.yml#L11